src/main/resources/assets/computercraft/lua/bios.lua

```diff
@@ -982,6 +982,8 @@ settings.set( "edit.default_extension", "lua" )
 settings.set( "paint.default_extension", "nfp" )
 settings.set( "lua.autocomplete", true )
 settings.set( "list.show_hidden", false )
+settings.set( "cctpm.directory", "/var/cctpm" )
+settings.set( "cctpm.repolist", "repolist|https://gitlab.com/Wilma456/cctpm/raw/master/repolist.txt" )
```

Comment:
Why does `repolist` occur in both the setting name and the value..?

Reply:
The repolist inside the value is for the type. There are 2 different types: repolist, which are a list of repos (on this way we can add repos without publishing a new Version) and repo, which are the direct link to a repo.